server/service/src/log.rs

```diff
@@ -0,0 +1,23 @@
+use repository::PaginationOption;
+use repository::{Log, LogFilter, LogRepository, LogSort, StorageConnectionManager};
+
+use super::{get_default_pagination, i64_to_u32, ListError, ListResult};
+
+pub const MAX_LIMIT: u32 = 1000;
+pub const MIN_LIMIT: u32 = 1;
+
+pub fn get_logs(
+    connection_manager: &StorageConnectionManager,
+    pagination: Option<PaginationOption>,
+    filter: Option<LogFilter>,
+    sort: Option<LogSort>,
+) -> Result<ListResult<Log>, ListError> {
+    let pagination = get_default_pagination(pagination, MAX_LIMIT, MIN_LIMIT)?;
```

Comment:
btw. we discussed to not limit the pagination on the service side and leave it to the caller, e.g. if you want to get any number of logs from another service in one call...